definitions/npm/express_v4.x.x/flow_v0.25.x-/express_v4.x.x.js

```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: any;
```

Comment:
@jeffmo In this case I think we should list fields that should be generic. For example:
***Request***
_body_
***Response***
_body_ in the _`send`_ method.

Also. I did some [tests](https://flowtype.org/try/#0MYGwhgzhAEBKCmBHArvCAXAPAFWgLmgHkAjAK3mHQD5oBvAKAEhiB7AEwE8DsBuegXz710HAA7xoACTAA7NiHgAnHDQC80ABSKkBBCjRZsVAJTRVNAG4sAlmyGhIMWC2ToJDaNFEsMKjaLB0AAsCDEVrGQBzABpoEKlZeSUVYwIrWzp6T08s6H4BenoZeAB3OBc3DWMAOm8MDQByAHoG2K1dJFRfAEFFRTAOTDCIyKoTMxpgFhkIFgVqkBZIrWrWTmNjHiA), and find out that you can't assign an `Array` to an `Object` field o__O
```
let a : Object = []; // empty array literal. This type is incompatible with object type.
```

not sure if this is a flowtype bug, but I think that if a request is a json, it can be an array.. BTW, if you use json middleware for array, you can receive all values, that are valid json.. So I don't think we have to constrain it to object.